lib/screens/listOfOrgs/listOfOrgsScreen.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:flutter/material.dart';
+import 'package:teesco/screens/listOfOrgs/widgets.dart';
+
+class ListOfOrganisations extends StatefulWidget {
+  @override
+  _ListOfOrganisationsState createState() => _ListOfOrganisationsState();
+}
+
+class _ListOfOrganisationsState extends State<ListOfOrganisations> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: appBar(),
```

Comment:
you need to create a separate widget only if the widget is very big
also, go through the appbar [documentation](https://api.flutter.dev/flutter/material/AppBar-class.html) and [examples](https://flutter.dev/docs/catalog/samples/basic-app-bar) for better understanding of how appbar should be used